app/timetables/models.py

```diff
@@ -39,3 +39,21 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
+    """
+    Model representing Dish.
+
+    A dish represents the actual food served as a given 
+    course under an option of a meal on a cycle day in a timetable. 
+    E.g, Coconut rice garnished with fish stew and chicken or just 
+    Ice-cream.
+    """
+
+    name = models.CharField(max_length=60, unique=True)
```

Comment:
Just got to know about this, please let's make it 255 instead of 256. At database-level, 1 byte is used to represent a 255-char-long field; 256 requires 2 bytes.